src/Context/Auth/Auth.js

```diff
@@ -29,7 +39,7 @@ const fetchLoginDetails = async (email,password) => {
     // saving the encodedToken in the localStorage
     
     localStorage.setItem("token", response.data.encodedToken);
-    localStorage.getItem("token", response.data.encodedToken);
+   // setAuthToken(localStorage.getItem("token"))
      navigate("/");
```

Comment:
You can extract localStorage as separate function

Reply:
okk sure